static/templates/task-feed/main.html

```diff
@@ -72,11 +72,13 @@ <h3 class="comment-sender">{{ item.comment.sender_alias }}</h3>
                 </div>
             </md-card>
         </div>
-        <md-card ng-if="!taskfeed.projects.length">
-            <md-card-content>
-                <h2 class="md-title">No free tasks available</h2>
-            </md-card-content>
-        </md-card>
+        <div ng-show="taskfeed.availableTasks">
```

Comment:
@maniksingh92, did you test this? I actually did this intentionally to get rid of the annoying flicker. So 'no more tasks left' only shows up once we we're sure there are none left.